lib/LangInterface.php

```diff
@@ -1256,16 +1259,74 @@ private static function get_new_source_id( string $type = 'post' ) : string {
 	 */
 	private static function translate_post_meta( int $post_id, string $new_lang, array $meta_keys_to_translate ) : bool {
 		global $wpdb;
-		$meta_keys_str = implode(
-			"','",
+		$simple_meta_keys = [];
+		$regex_meta_keys  = [];
+		$like_meta_keys   = [];
+
+		foreach ( $meta_keys_to_translate as $meta_key => $meta_data ) {
+			if ( ! $meta_data instanceof TranslationKey ) {
+				$simple_meta_keys[] = esc_sql( $meta_key );
+				continue;
+			}
+
+			if ( $meta_data instanceof RegexTranslationKey ) {
+				if ( $meta_data->has_sql_like() ) {
+					$like_meta_keys[] = esc_sql( $meta_data->get_sql_like() );
+					continue;
+				}
+
+				$regex_key = $meta_data->get_key();
```

Comment:
It makes more sense to have this in the `RegexTranslationKey` class interface. There we can call for the correct string depending on what is needed. We could add a param to this method, to include or exclude the `/` characters.